scripts/generate_aviti_run_manifest.py

```diff
@@ -189,7 +189,9 @@ def get_manifests(process: Process, manifest_root_name: str) -> list[tuple[str,
                     row["Index1"], row["Index2"] = idx
                 else:
                     row["Index1"] = idx
-                    row["Index2"] = ""
+                    # Assume long idx2 from recipe + no idx2 from label means idx2 is UMI
+                    if int(process.udf.get("Index read 2", 0)) > 12:
+                        row["Index2"] = "N" * int(process.udf["Index read 2"])
```

Comment:
Do we still need to keep the placeholder `row["Index2"] = ""` in an `else`?

Reply:
That's right, good catch 🙏 
[05d37d4](https://github.com/SciLifeLab/scilifelab_epps/pull/367/commits/05d37d48f06cfc701d7c7ff22274cf8f907eb173)